### ISSUE TITLE
sorry

### ISSUE THREAD

**Issue by wuzelei123** [amirhertz/pointgmm#2]
Why isn't the loss function of your model decreasing?

**Comment by wuzelei123** [comment]
The loss function  became negative，when I trained your model for 25 epoches. It seems have some problem in your loss function design.
